Include/cpython/dictobject.h

```diff
@@ -81,3 +81,6 @@ typedef struct {
 
 PyAPI_FUNC(PyObject *) _PyDictView_New(PyObject *, PyTypeObject *);
 PyAPI_FUNC(PyObject *) _PyDictView_Intersect(PyObject* self, PyObject *other);
+
+PyAPI_FUNC(Py_ssize_t) _PyDict_GetItemHint(PyDictObject *, PyObject *, Py_ssize_t, PyObject **);
```

Comment:
I think this API should be private; only Python core use it. Move this API right after `_PyDict_LoadGlobal`, and remove `PyAPI_FUNC`.

Reply:
Done